src/getConfig.ts

```diff
@@ -0,0 +1,31 @@
+import { exec } from 'child_process';
+
+interface Config {
+  project: {
+    android: {
+      packageName: string;
+    };
+  };
+  root: string;
+}
+
+function getConfig(): Promise<Config | null> {
+  return new Promise((resolve, reject) => {
+    exec('npx react-native config', (error, stdout, stderr) => {
+      if (error) {
+        console.error(`exec error: ${error}`);
+        reject(error);
+        return;
+      }
+
+      resolve(JSON.parse(stdout));
+
+      if (stderr) {
+        resolve(null);
+        throw new Error(stderr);
+      }
```

Comment:
@szymonrybczak why exactly did we do that here? That on stderr we resolve but throw an error?

(cc @kirillzyusko)

Reply:
I'm entirely sure what do you mean? Can you specify?

Reply:
I mean why would we "resolve" on an stderr output but then throw an error?
![Screenshot 2024-06-27 at 10 27 07](https://github.com/margelo/react-native-release-profiler/assets/16821682/c16a9eb6-c1d1-47f2-a5f9-58881b0d624a)

Reply:
instead of rejecting on an stderr?

Reply:
If there's something inside `stderr` there's no config, soo it was more logical to me to just return `null`. Not strong opinion here tho.